CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [2.6.0] - 2024-12-03
+This update includes Extensions, a mechanism to handle extensions on a device for custom data and/or reporting outside the firmware update mechanism. These are
+isolated from the socket as to not get in the way of potential firmware updates and can be defined in external libs as well as this one by implementing the `NervesHubLink.Extensions` behavior.
```

Comment:
```suggestion
## [2.6.0] - 2024-12-03

This update includes Extensions, a mechanism to handle extensions on a device
for custom data and/or reporting outside the firmware update mechanism. These
are isolated from the socket as to not get in the way of potential firmware
updates and can be defined in external libs as well as this one by implementing
the `NervesHubLink.Extensions` behavior. Extensions are controlled by NervesHub
and can be adjusted at the product or device level for granular control.
```
Adjusted spacing, kept to 80 character line limit, and added note about NervesHub controlling